packages/injected/src/types.ts

```diff
@@ -40,7 +40,8 @@ export enum ProviderIdentityFlag {
   OneInch = 'isOneInchIOSWallet',
   Tokenary = 'isTokenary',
   Tally = 'isTally',
-  BraveWallet = 'isBraveWallet'
+  BraveWallet = 'isBraveWallet',
+  Rabby = 'isRabby',
```

Comment:
We have `trailingComma` disabled. I would run `yarn format` just to make sure your files are correctly formatted
```suggestion
  Rabby = 'isRabby'
```